### DISCUSSION TITLE
Repeated VLAN Group name but different scope - shouldn't this be possible?

### DISCUSSION THREAD

**Discussion by tfboy** [netbox-community/netbox#11789] [Q&A]
I'm reorganising my VLAN Groups in Netbox and allocating minimum and maximum VLAN IDs.
So far so good.
But, I think there's a bug / inconsistency in with the scope concept.
I thought you could reuse the same name for a VLAN Group if it's in a different scope. But it appears this isn't the case.
I have a VLAN Group called "front" with VLAN IDs from 200 to 299 which has a scope limited to Site A.
I want to create a new VLAN Group also called "front" with VLAN IDs from 300 to 399 which has a scope limited to Site B.
This isn't possible.
When I try, I get an error message: "Record 1 group: "front" is not a unique value for this field; multiple objects were found".

So have I misunderstood the concept of a scope or is there a bug ?

**Comment by tfboy** [comment]
Sorry, to clarify, I was attempting to import via a CSV.
If I go to add an individual VLAN manually, then it becomes obvious I've misunderstood the concept of scope.
I'll just rename my VLAN Groups so they carry unique names.

**Comment by jeremystretch** [comment]
VLAN groups in different scopes can have the same name, but when importing VLANs, you need to uniquely identify each group to which a VLAN is being assigned. Just specifying the name isn't sufficient, because you have multiple VLAN groups with the same name, hence the error message. You can specify the VLAN groups by their numeric IDs instead by renaming the CSV column header to `group.id`.
